src/algorithms/calorimetry/CalorimeterHitReco.cc

```diff
@@ -172,6 +172,11 @@ void CalorimeterHitReco::process(
             continue;
         }
 
+        if (rh.getAmplitude() > m_cfg.capADC) {
+            error("Encountered hit with amplitude {} outside of ADC capacity {}", rh.getAmplitude(), m_cfg.capADC);
```

Comment:
(I wonder if this makes more sense to put in Digi.)

Reply:
I think it should be in both. This is intended as the check on the digitization.